### ISSUE TITLE
Analyse CellPlex fast files

### ISSUE THREAD

**Issue by kikegoni** [pachterlab/kallisto#307]
Hey,

First of all thanks for developing the kallisto bus for scRNA-seq, it's proven to be very useful. 

I was wondering whether is possible to process fast files processed with CellPlex technology (Cell Multiplexing, from 10X). 

For example, if we download the fast files from this dataset https://support.10xgenomics.com/single-cell-gene-expression/datasets/6.0.0/SC3_v3_NextGem_DI_CellPlex_Mouse_PBMC_10K_Multiplex we can see three fast ids:
- 1 referring to the library (get file) and the other two referring to the 2 CMOs that mark each sample.

You can see this here:

10x has modified the classical "cellranger count" processing they do traditionally https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/multi and they do now "cell ranger multi". 

I was wondering whether kallisto bus is able to process the fast files produced by this way and if that's the case, they should just be run on the .gex directory, right?

Thank you very much for any help you can provide here,

Best,

Kike

**Comment by sbooeshaghi** [comment]
Yes this is possible using the `kite` workflow implemented in kb. Soon we will be releasing a manuscript that describes the method in detail but in the meantime you can follow along on this repo: https://github.com/sbooeshaghi/BMGP_2020/
